apps/perf-test-react-components/src/scenarios/FluentProviderWithTheme.tsx

```diff
@@ -5,9 +5,11 @@ import { teamsLightTheme } from '@fluentui/react-theme';
 const LayoutShift: React.FunctionComponent = ({ children }) => {
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28887&commitId=9d9695a70487e3f6bfb56108c08d5c2a7cea12b4&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.default.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>